fjord/feedback/views.py

```diff
@@ -135,10 +142,10 @@ def _handle_feedback_post(request, locale=None, product=None,
     data = form.cleaned_data
 
     description = data.get('description', u'').strip()
+
     if not description:
-        # If there's no description, then there's nothing to do here,
-        # so thank the user and move on.
-        return HttpResponseRedirect(reverse('thanks'))
+        # If there's no description populate a generic one
+        description = u'Sentiment: %s' % (('Sad', 'Happy')[data['happy']])
```

Comment:
Input gathers feedback for sentiment analysis which requires text content. In order to accept feedback without descriptions, we'd need to do a major overhaul of Input or create a parallel feedback system. That's a big project and has a lot of ramifications.

Thinking about the options a bit, we can't send them to the apple store *and* also have them leave feedback in Input. We also can't forward feedback from Input to the apple store. Given that we want happy feedback in the apple store, I think the only option here is if the user says "happy", we give them an interstitial page that links to the apple store and that's the end of the flow. The form shouldn't POST anything to Input. The user never sees the "thank you" page.